std/functional.d

```diff
@@ -656,14 +656,6 @@ unittest
 $(LINK2 http://en.wikipedia.org/wiki/Partial_application, Partially
 applies) $(D_PARAM fun) by tying its first argument to $(D_PARAM arg).
 
-Example:
-
-----
-int fun(int a, int b) { return a + b; }
-alias partial!(fun, 5) fun5;
-assert(fun5(6) == 11);
-----
-
 Note that in most cases you'd use an alias instead of a value
 assignment. Using an alias allows you to partially evaluate template
 functions without committing to a particular type of the function.
```

Comment:
Does this paragraph still make sense now that the unittest example comes after it?